scripts/create-jazz-bar.ts

```diff
@@ -56,7 +56,8 @@ const jazzbar: JazzbarVenue = {
   code_of_conduct_questions,
 };
 
-const venueId = name.replace(/\W/g, "").toLowerCase();
+const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
+const venueId = name.replace(INVALID_SLUG_CHARS_REGEX, "").toLowerCase();
```

Comment:
Could we re-use the code from `utils/venue.js`?

Reply:
No. Doing so is OOS.

One is run in the browser, another one is run in the google cloud, third one is run in the terminal.

Reply:
Agreed that this is OOS.

For my own education about this project: would it be hard to extract the slug logic out to one module and have all three areas use the same module?

Reply:
Starting from the current state: yes.
Starting with a blank slate: maybe, I don't know  (haven't configured TS to output code for one, let alone 3 environments).